config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml

```
@@ -39,7 +39,7 @@ metadata:
     operatorframework.io/arch.ppc64le: supported
     operatorframework.io/arch.s390x: supported
     operatorframework.io/os.linux: supported
-  name: cert-manager-operator.v1.15.0
+  name: cert-manager-operator.v1.16.0
```

Can you change this to

```suggestion
  name: cert-manager-operator
```

That way we don't need to update this field in the CSV of `config/*/` for future bumps.

---

I'm afraid we still need to keep `.v1.16.0` for consistency, since it's also be coded in L109 https://github.com/openshift/cert-manager-operator/blob/c1fb4fcbf97e42f1a0048735601f2f6136573d2b/config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml#L109